app/controllers/events_controller.rb

```diff
@@ -35,14 +39,16 @@ def index
 
   def create
     conference_room_id = event_params[:conference_room_id]
-    google_event_params = GoogleEvent.process_params(event_params)
-    data = GoogleEvent.create(session[:credentials], conference_room_id, google_event_params)
+    google_event_params = GoogleCalendar::GoogleEvent.process_params(event_params)
+    google_event = GoogleCalendar::GoogleEvent.new(session[:credentials], session[:email])
+    data = google_event.create(conference_room_id, google_event_params)
```

Comment:
you create something for particular conference_room_id, so may by would be better to have it inside class as instance variable?